scripts/dts/gen_defines.py

```diff
@@ -110,6 +110,28 @@ def main():
         write_chosen(edt)
         write_global_compat_info(edt)
 
+    with open(args.device_header_out, "w", encoding="utf-8") as dev_header_file:
```

Comment:
Would you mind making a separate function for this?

Reply:
Done.